buyer/issue/README.md

```diff
@@ -78,6 +78,7 @@ The `supplierAccountNumber` should be set on forehand in the Tradecloud connecti
 
 * `properties`: are key-value based custom fields. You can user as many as needed, but too many will clutter the portal. Use `\n` for a new line in the value.
 * `notes`: are simple custom fields. You can user as many as needed, but too many will clutter the portal. Use `\n` for a new line.
+* `labels`: shipment labels when shipped per order. Please note the practicable number of labels is dependent on the supplier.
```

Comment:
Labels are not intended for shipments but for Value Added Services. These are services that the supplier does for the buyer. Erics labels the items for the allocation in the warehouse of Damen.

VAS could be something like packaging, repackaging, Labeling.

Reply:
@Sjorstradecloud Replaced "shipment" by "value-added services"